src/resolution/__init__.py

```
@@ -34,6 +38,7 @@ class ComplianceLevel(Enum):
 @dataclass
 class ConstitutionalValidationResult:
     """Result of constitutional validation"""
+
     overall_score: float  # 0.0 to 1.0
     compliance_level: ComplianceLevel
     detailed_results: List[ComplianceResult]
```

Flake8 (7.3.0)

[error] undefined name 'ComplianceResult'

(F821)